.vscode/launch.json

```diff
@@ -7,7 +7,7 @@
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
-            "args": ["--extensionDevelopmentPath=${workspaceRoot}" ],
+            "args": ["--extensionDevelopmentPath=${workspaceRoot}", "/home/jholzl/Projects/lean-scratch/coinduction.lean"],
```

Comment:
Ah sorry. While it opens the last folder, I had the problem that it also opened a new file. And not the file I used to test errors.